tbx/blog/models.py

```diff
@@ -42,12 +43,25 @@ def taxonomy_slugs(self):
 
     @property
     def blog_posts(self):
+        prefetch_author_images = models.Prefetch(
+            "authors__author__image",
+            queryset=CustomImage.objects.prefetch_renditions(
+                "format-webp|fill-72x72",
```

Comment:
It would be helpful wherever you are prefetching images with a specific size, to add a comment in the equivalent template, so that if the image renditions need to be updated, they are also updated here. Andy also mentioned that you don't necessarily need to fetch each specific rendition - apparently there is not that much overhead in prefetching all the possible renditions for an image, and that way it is more future-proof against changes in the template.

Reply:
done